test/common-utils/devcontainer-info/Dockerfile

```
@@ -0,0 +1,3 @@
+FROM mcr.microsoft.com/devcontainers/base:debian
```

```suggestion
FROM ubuntu:focal
```

The latest MCR image is built with `common-utils:1`, hence the `devcontainer-info` command is correctly configured. The newly added tests in this PR still pass against it without the `chmod +rx` changes.

Can we change the image here to avoid the false positives? thanks!

---

Have a PR out to make that version bump switch for the images. But I'll be waiting to merge that until this PR gets merged.

https://github.com/devcontainers/images/pull/307/files#r1065017365